cocos/misc/renderer.ts

```diff
@@ -161,9 +161,9 @@ export class Renderer extends Component {
      * new material instance will be created automatically if the sub-model is already using one.
      * @zh 设置指定子模型的 sharedMaterial，如果对应位置有材质实例则会创建一个对应的材质实例。
      */
-    public setMaterial (material: Material | null, index: number): void {
+    public setSharedMaterial (material: Material | null, index: number): void {
```

Comment:
Shall we keep and deprecate `setMaterial` for sometime, so user can change their code?

Reply:
Updated